src/open_inwoner/openklant/constants.py

```diff
@@ -17,3 +17,20 @@ def safe_label(cls, value, default=""):
             if default:
                 return default
             return str(value).replace("_", " ").title()
+
+
+class ContactmomentKanaalEsuite(models.TextChoices):
```

Comment:
How sure are we that this enumeration is the same across multiple eSuite instances? As with most things eSuite, this is likely something which is configurable...

Reply:
Discussed, we'll replace the whitelist containing hard-coded strings with a blacklist to be entered by the admin (cf. https://taiga.maykinmedia.nl/project/open-inwoner/task/2724)